app/geo/gui/src/ts/twxplore/gui/geo/actions/map/AllFiltersSetAction.ts

```diff
@@ -0,0 +1,17 @@
+import {Action} from "redux-actions";
+import {FeatureType} from "../../api/graphqlGlobalTypes";
+
+export type ALL_FILTERS_SET = "ALL_FILTERS_SET";
+export const ALL_FILTERS_SET: ALL_FILTERS_SET = "ALL_FILTERS_SET";
+
+export interface AllFiltersSetAction
+  extends Action<{featureType: FeatureType}> {
+  type: ALL_FILTERS_SET;
+}
+
+export const allFiltersSet = (
```

Comment:
Nit: usually action names start with a verb, like method names. Fine to leave this as it is.